book/security.md

```diff
@@ -405,184 +406,192 @@ them steal cookie values...
 
 
 
-Cross-site scripting
-====================
+Cross-site requests
+===================
 
-With cookies accessible from JavaScript, any scripts run on our server
-could, in principle, read the cookie value. This might seem
-benign---doesn't our server only run `comment.js`? Well...
+Imagine that your browser is logged in to your bank, and than an
+attacker wants to know your (private) bank balance. Already, our
+browser stores different cookies for each site. Because of this, it
+won't just send the bank cookie to an attacker's site. But what if the
+attacker is clever?
 
-A web service needs to defend itself from being *misused*. Consider
-the code in our guest book that outputs guest book entries:
+The easiest way for an attacker to steal your private data is to ask
+for it. Of course, there's no API in the browser for a website to ask
+for another website's cookies. But there _is_ an API to make requests
+to another website. It's called `XMLHttpRequest`.[^weird-name]
 
-``` {.python file=server indent=8}
-out += "<p>" + entry + " <i>by " + who + "</i></p>"
-```
+[^weird-name]: It's a weird name! Why is `XML` capitalized but not
```

Comment:
Are you sure it can handle non-HTTP requests? I'm not sure that's correct.

Reply:
Add: "The XML name comes from the historical origins of the API as part of the Outlook Web Access project at Microsoft (https://en.wikipedia.org/wiki/XMLHttpRequest)

Reply:
I did edit this footnote to include the history.

I'll remove the HTTP comment, I agree it's a stretch.